docs/transport/amqp_bunny.md

```diff
@@ -267,4 +268,17 @@ $queue = $context->createQueue('aQueue');
 $context->purgeQueue($queue);
 ```
 
+## Enable publisher confirms
+
+If you use RabbitMQ and want to be sure your messages are received and persisted to the disk,
+you can turn on the [publisher confirms](https://www.rabbitmq.com/confirms.html#publisher-confirms).
+
+This way, after each published message, your PHP script will wait up to specified number
+of seconds to get confirmation (ACK frame) from RabbitMQ.
+
+```php
+<?php
+$factory = new AmqpConnectionFactory('amqp:?publisher_confirm_timeout=3');
```

Comment:
Publsher confirms is a RabbitMQ extension, works only for it. Should be prefixed `rabbitmq_`.